sdk/identity/identity/src/msal/flows.ts

```diff
@@ -18,6 +18,7 @@ export interface MsalFlowOptions {
   authorityHost?: string;
   authenticationRecord?: AuthenticationRecord;
   disableAutomaticAuthentication?: boolean;
+  allowPiiLogging?: boolean;
```

Comment:
Nit: For ease of readability, I would recommend moving this up to appear right after the logger as the two are related